src/main/java/org/openstreetmap/atlas/checks/validation/intersections/BuildingRoadIntersectionCheck.java

```diff
@@ -29,16 +31,25 @@
 
 /**
  * Flags buildings that intersect/touch centerlines of roads. This doesn't address cases where
- * buildings get really close to roads, but don't overlap them.
+ * buildings get really close to roads, but don't overlap them. The configurable value
+ * "car.navigable" can be set to true or false, depending on which the validity of the intersecting
+ * highways will be checked. The default value of "car.navigable" is set to true. If set to true,
+ * the intersecting highways will be checked for tags in the enum set "CAR_NAVIGABLE_HIGHWAYS", else
+ * checked for tags in "CORE_WAYS" enum set in the {@link HighwayTag} class.
  *
- * @author mgostintsev
+ * @author mgostintsev, sayas01
  */
 public class BuildingRoadIntersectionCheck extends BaseCheck<Long>
 {
+    private static final String BUILDING_ROAD_INTERSECTION_INSTRUCTION = "Building (id-{0,number,#}) intersects road (id-{1,number,#})";
+    private static final String SERVICE_ROAD_INTERSECTION_INSTRUCTION = "Building (id-{0,number,#}) intersects road (id-{1,number,#}), which is a SERVICE road. Please verify whether the intersection is valid or not.";
     private static final List<String> FALLBACK_INSTRUCTIONS = Arrays
-            .asList("Building (id-{0,number,#}) intersects road (id-{1,number,#})");
+            .asList(BUILDING_ROAD_INTERSECTION_INSTRUCTION, SERVICE_ROAD_INTERSECTION_INSTRUCTION);
     private static final String INDOOR_KEY = "indoor";
     private static final String YES_VALUE = "yes";
+    private static final Predicate<Edge> highwayServiceTag = edge -> Validators.isOfType(edge,
```

Comment:
Can you snake case + caps lock this to fit wit the style for static final constants.